src/components/Home.jsx

```diff
@@ -0,0 +1,20 @@
+import { useNavigate } from "react-router-dom";
+
+export const Home = () => {
+  let navigate = useNavigate();
+  const routeChange = () => {
+    let path = `reviews`;
+    navigate(path);
+  };
+
+  return (
+    <div className="Home">
+      <h2>Welcome!</h2>
+      <h3>Pick a category</h3>
+      <h4>or</h4>
+      <button className="reviewListButton" onClick={routeChange}>
```

Comment:
keeping in mind accessibility best practices, we use links to navigate places, and buttons to implement behaviour; consider refactoring to be more accessible